nox/virtualenv.py

```diff
@@ -31,6 +31,12 @@
 _SYSTEM = platform.system()
 
 
+class InterpreterNotFound(EnvironmentError):
```

Comment:
```
$ python2.7 -c 'print(EnvironmentError)'
<type 'exceptions.EnvironmentError'>
$ python3.7 -c 'print(EnvironmentError)'
<class 'OSError'>
```

Reply:
Fixed, good catch. (OSError is fine, but let's use the non-deprecated name).